types/geobuckets/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for geobuckets 0.0
+// Project: https://github.com/mtralka/GeoBuckets#readme
+// Definitions by: Dmitry Baranov <https://github.com/DmitryBaranovAndreevich>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ If this module is a UMD module that exposes a global variable 'myLib' when
+ *~ loaded outside a module loader environment, declare that global here.
+ *~ Otherwise, delete this declaration.
+ */
+
```

Comment:
```suggestion
/*~ If this module is a UMD module that exposes a global variable 'myLib' when
 *~ loaded outside a module loader environment, declare that global here.
 *~ Otherwise, delete this declaration.
 */

```
these are template guidelines, no need for them in actual code